### ISSUE TITLE
About number of epochs/iters

### ISSUE THREAD

**Issue by PLoic** [charlesCXK/TorchSemiSeg#19]
Hi @charlesCXK, I was reading your paper and code, and I got a question concerning the numbers of epochs (max number of iters), did you choose it arbitrary ? Or do you take the fully supervised number of iters from deeplabv3+ as a starting point and adapt it to the various splits ?

I hope my question is clear :)

**Comment by charlesCXK** [comment]
Hi, the **max number of epochs** is not equal to the **max number of iters** in our code. We set different **epochs** for different data splits, but the total iterations (**max number of iters**) of different experiments are kept the same. (Due to the numbers of unlabeled images  are different for different data splits, the iters in one epoch are different as well.)

**Comment by PLoic** [comment]
Thanks for your answer. I have also seen in your code you use a base_lr of 0.02 (city expe) and in your paper you mentioned a base_lr of 0.04. Is it an error in the config files ? Also, did you need to multiply the base_lr by the number of used GPU in distributed train ?

Thanks again for your answer  !

**Comment by charlesCXK** [comment]
Hi, the learning rate reported in our paper is described as `For the supervised baseline trained on the full training set`. This means the lr=0.04 is set when the batch size of the labeled data is 16.

The base_lr needs to multiply by the number of GPUs on VOC. However, for the Cityscpaes dataset, we recommend not to do this.
